consortia/argocd-app-templates/appsetup-stable.yaml

```diff
@@ -23,19 +23,230 @@ metadata:
+              appmarketplace:
+                logging:
+                  default: "Debug"
+                  offersLibrary: "Debug"
+                healthChecks:
+                  startup:
+                    tags:
+                    - name: "HEALTHCHECKS__0__TAGS__0"
+                      value: "keycloak"
+                    - name: "HEALTHCHECKS__0__TAGS__1"
+                      value: "portaldb"
+                swaggerEnabled: true
+              portalmigrations:
+                logging:
+                  default: "Debug"
+                seeding:
+                  testDataEnvironments: "consortia"
+              notification:
+                healthChecks:
+                  startup:
+                    tags:
+                    - name: "HEALTHCHECKS__0__TAGS__0"
+                      value: "keycloak"
+                    - name: "HEALTHCHECKS__0__TAGS__1"
+                      value: "portaldb"
+                logging:
+                  default: "Debug"
+                swaggerEnabled: true
+              services:
+                logging:
+                  default: "Debug"
+                  offersLibrary: "Debug"
+                healthChecks:
+                  startup:
+                    tags:
+                    - name: "HEALTHCHECKS__0__TAGS__0"
+                      value: "keycloak"
+                    - name: "HEALTHCHECKS__0__TAGS__1"
+                      value: "portaldb"
+                swaggerEnabled: true
+              processesworker:
+                logging:
+                  default: "Debug"
+                  processesLibrary: "Debug"
+                  bpdmLibrary: "Debug"
+                  clearinghouseLibrary: "Debug"
+                  custodianLibrary: "Debug"
+                  sdfactoryLibrary: "Debug"
+                  offerProvider: "Debug"
+                bpdm:
+                  clientId: "<path:portal/data/processes-worker#bpdm-client-id>"
+                  clientSecret: "<path:portal/data/stable/processes-worker#bpdm-client-secret>"
+                clearinghouse:
+                  clientId: "<path:portal/data/stable/processes-worker#clearinghouse-client-id>"
+                  clientSecret: "<path:portal/data/stable/processes-worker#clearinghouse-client-secret>"
+                custodian:
+                  clientId: "<path:portal/data/processes-worker#custodian-client-id>"
+                  clientSecret: "<path:portal/data/stable/processes-worker#custodian-client-secret>"
+                sdfactory:
+                  issuerBpn: "BPNL00000003CRHK"
+                  clientId: "<path:portal/data/processes-worker#sdfactory-client-id>"
+                  clientSecret: "<path:portal/data/stable/processes-worker#sdfactory-client-secret>"
+                offerprovider:
+                  clientId: "<path:portal/data/processes-worker#offerprovider-client-id>"
+                  clientSecret: "<path:portal/data/stable/processes-worker#offerprovider-client-secret>"
```

Comment:
there was a new secret introduced for onboarding-service-provider

Reply:
thanks, I added https://github.com/eclipse-tractusx/portal-cd/pull/91/files#diff-655f983f01cd391de5332255653b7b05cb3be87d3f9a6876ec90276e494b6287R163

Reply:
I did forget [this](https://github.com/eclipse-tractusx/portal-cd/pull/91/commits/9084da642d3437f4ce916a0934bebb162b244eab#diff-655f983f01cd391de5332255653b7b05cb3be87d3f9a6876ec90276e494b6287R240) earlier, should be fine now